### PR TITLE
release(v1.0.0): prepare release

### PR DESCRIPTION
**Pull request by Unix4ever**
This is the official v1.0.0 release.

Signed-off-by: Artem Chernyshev <artem.chernyshev@talos-systems.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,704 @@
...
+
+Main changes UX-wise:
+- all clusters are now placed in the context dropdown: both kubeconfig clusters and CAPI clusters.
+- now it is possible to see Kubernetes upgrades history (dropped after theila restart as of now).
+- pod information has got more details.
+- servers list is now also available for CAPI clusters.
+- now the UI provides summary page for cluster resources usage.
+
+
+### Contributors
```
Comment (frezbo) [inline CHANGELOG.md:23]: is the contributor list here correct, seems like the one for talos?

Comment (Unix4ever) [reply CHANGELOG.md:23]: I think it gathers contributors from all deps, and Talos is one of them, so I guess it's correct.

**Comment by andrewrynhard** [comment]
Weren't we going to say 0.1?

**Comment by smira** [comment]
> Weren't we going to say 0.1?

I vote to stay in sync with Talos, but I think it's not a big deal either way

**Comment by Unix4ever** [comment]
I've heard 1.0 from you, but maybe I was wrong :)

**Comment by Unix4ever** [comment]
we already had 0.1 btw, so it should have been 0.2

**Comment by Unix4ever** [comment]
I will create 0.2 release instead then.

**Pull request closed without merging**